app/models/service.rb

```diff
@@ -18,6 +20,10 @@ def self.search_by_name(query)
   end
 
   def service_rating
+    service_rating_get
```

Comment:
This is probably due to my inexperience with RoR: what does this do?

Reply:
I think I did that because it seemed that we were using a instance method directly in the view before, which is technically against convention, so I just dumped it in an attribute accessor.

Reply:
Think I also had the intention of making service_rating_get a private method.